### PR TITLE
Secure Practice MailRisk data connector

### PR DESCRIPTION
**Pull request by ingin97**
Required items, please complete
   
   Change(s):
   - Update solution_id/offer_id and support address

   Reason for Change(s):
   - Correct mistakes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Solutions/MailRisk/Package/2.0.2.zip
Comment (v-vdixit) [inline Solutions/MailRisk/Package/2.0.2.zip:1]: Hi @ingin97 Please ensure that the package created only has the latest CreateUI and maintemplate files and not the older packages, currently I can see this zip file contains older packages too, please check on this, thanks!
image

Comment (v-vdixit) [reply Solutions/MailRisk/Package/2.0.2.zip:1]: Hi @ingin97 thanks for updating package, your approach is correct the recent version of packaging tool includes older packages by default, we will work on updating this in the next version.

File: Solutions/MailRisk/Package/mainTemplate.json [hunk 1]
```diff
@@ -30,9 +30,9 @@
     }
   },
   "variables": {
-    "solutionId": "securepractice.azure-sentinel-solution-mailrisk",
+    "solutionId": "securepractice.microsoft-sentinel-solution-mailrisk",
```
Comment (v-vdixit) [inline Solutions/MailRisk/Package/mainTemplate.json:33]: Hi @ingin97 we are trying to deploy the solution but it is not reflecting post deployment, can you please confirm why this offer id has been updated, thanks!

Comment (ingin97) [reply Solutions/MailRisk/Package/mainTemplate.json:33]: Hi @v-vdixit, the solution is not yet published, and the offer in the Partner for publishing somehow got mismatched with the publisher ID, we were adviced from a Partner Program Manager that the simplest fix was to change the offer name in the solution id for it to match properly.

**Review by v-vdixit — APPROVED** [review]

**Pull request merged**